Lib/test/support/__init__.py

```diff
@@ -2118,3 +2119,19 @@ def clear_ignored_deprecations(*tokens: object) -> None:
     if warnings.filters != new_filters:
         warnings.filters[:] = new_filters
         warnings._filters_mutated()
+
+
+# This number should be CPython internal "hot" count times 2.
+CPYTHON_WARMUP_COUNT = 8*2
```

Comment:
I'm surprised. In my reproducer, it fails at the 7th iteration: https://bugs.python.org/issue46465#msg411220

Reply:
Why "x2"? Testing adapative opcode would be enough, so 8 is enough, no?

Reply:
My paranoia :). But yeah I agree that 16x is overkill, I've reduced it to 9.

I've been bitten by such things many times (this bpo is the 3rd or 4th time that some tests passed normally but fails on -R 3:3 due to my errors when implementing specialized opcodes). Sometimes they need 1 or 2 tests more to actually fail - I had a memory corruption in test_descr before that only triggered after the 2nd specialized opcode run.